src/server.cpp

```diff
@@ -291,26 +288,18 @@ Server::Server(
 	}
 	infostream << std::endl;
 	// Load and run "mod" scripts
-	for (std::vector<ModSpec>::iterator i = m_mods.begin();
-			i != m_mods.end(); ++i) {
-		const ModSpec &mod = *i;
+	for (std::vector<ModSpec>::iterator it = m_mods.begin();
+			it != m_mods.end(); ++it) {
+		const ModSpec &mod = *it;
```

Comment:
I like it or iter for iterators because it's immediately clear that the variable is an iterator.  I approve of this change too.

Really, I like anything that makes the code clearer.